src/xml/namespace.cr

```diff
@@ -4,8 +4,8 @@ struct XML::Namespace
   def initialize(@document : Node, @ns : LibXML::NS*)
   end
 
-  def hash
-    object_id
+  def hash(hasher)
+    hasher << object_id
```

Comment:
ops, missed it when changed there.
it doesn't affect correctness, but for performance it is better to change to `raw`.
thanks.